index.js

```diff
@@ -107,26 +107,28 @@ async function authorize(options) {
   return fetch(`${ims}/ims/exchange/jwt/`, postOptions)
     .catch(e => throwRequestFailedError(e.message))
     .then(res => {
-      if (res.ok) {
-        return res.json();
-      } else {
-        throwRequestFailedError(res.statusText);
```

Comment:
delete the `throwRequestFailedError` here and let the details emerge near lines 123-130